### PR TITLE
Add option to configure .peppercache file

### PR DESCRIPTION
**Pull request by chnyda**
Enable cache file configuration using -x option

### CODE REVIEW

File: pepper/cli.py [hunk 1]
```diff
@@ -431,15 +438,15 @@ def run(self):
                 api.req('/stats')
             except Exception as e:
                 if e.args[0] is not 2:
-                    logger.error('Unable to load login token from ~/.peppercache '+str(e))
+                    logger.error('Unable to load login token from {} {}'.format(token_file, str(e)))
```
Comment (whiteinge) [inline pepper/cli.py]: Will you please index these substitutions so it works on 2.6?

File: pepper/cli.py [hunk 2]
```diff
@@ -431,15 +438,15 @@ def run(self):
...
                 auth = api.login(*self.parse_login())
                 try:
                     oldumask = os.umask(0)
                     fdsc = os.open(token_file, os.O_WRONLY | os.O_CREAT, 0o600)
                     with os.fdopen(fdsc, 'wt') as f:
                         json.dump(auth, f)
                 except Exception as e:
-                    logger.error('Unable to save token to ~/.pepperache '+str(e))
+                    logger.error('Unable to save token to{} {}'.format(token_file, str(e)))
```
Comment (whiteinge) [inline pepper/cli.py]: Index subs here too.

**Review by whiteinge — CHANGES_REQUESTED** [review]
Great addition! Thanks. Two small changes for 2.6 compat if you can.

File: pepper/cli.py (hunk 1 above)
Comment (chnyda) [reply pepper/cli.py]: Thanks for your quick review. I update the PR now.

**Review by whiteinge — APPROVED** [review]
Thanks!

**Pull request merged**